src/components/Navbar.js

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { MdClose } from 'react-icons/md';
+import { FiMenu } from 'react-icons/fi';
+
+const Navbar = () => {
+  const [navbarOpen, setNavbarOpen] = useState(false);
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      text: 'Home',
+    },
+    {
+      id: 2,
+      path: '/about',
+      text: 'About',
+    },
+  ];
+
+  const handleToggle = () => {
+    setNavbarOpen((prev) => !prev);
+  };
```

Comment:
I see that this usage of the `useState` set function differs from that in `InputTodo#onChangeHandler` in that one uses a callback to get a previous state and the other doesn't. What's the behavior of this callback?
If it is like that of `this.setState` on class components then please avoid using the value of your state variable in the call to the setter method (since it's unreliable).